### ISSUE TITLE
[qt] Offline database test throwing with Qt 5.10

### ISSUE THREAD

**Issue by tmpsantos** [mapbox/mapbox-gl-native#11295]
It is crashing with an uncaught exception because the query binding is not working.

```
$ BUILDTYPE=Release make run-qt-test-OfflineDatabase.UpdateMetadata -j8
platform/linux/ninja  -j4 -C build/qt-linux-x86_64/Release mbgl-test
ninja: Entering directory `build/qt-linux-x86_64/Release'
[4/4] Automatic moc and rcc for target mbgl-core
build/qt-linux-x86_64/Release/mbgl-test --gtest_catch_exceptions=0 --gtest_filter=OfflineDatabase.UpdateMetadata
executable: /usr/bin/nodejs
Note: Google Test filter = OfflineDatabase.UpdateMetadata
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from OfflineDatabase
[ RUN      ] OfflineDatabase.UpdateMetadata
terminate called after throwing an instance of 'mapbox::sqlite::Exception'
  what():  near "id": syntax error Unable to execute statement

```

Tested against the Qt tag `v5.10.1`.

/cc @brunoabinader

**Comment by brunoabinader** [comment]
FWIW this still happens to me after https://github.com/mapbox/mapbox-gl-native/pull/11303

**Comment by brunoabinader** [comment]
Fix in #11314.
